src/Workspaces/Core/Portable/FindSymbols/FindReferences/DependentProjectsFinder.cs

```diff
@@ -226,26 +238,29 @@ private static async Task AddSubmissionDependentProjectsAsync(Solution solution,
             // search only submission project
             foreach (var projectId in solution.ProjectIds)
             {
-                var project = solution.GetProject(projectId);
+                var project = solution.GetRequiredProject(projectId);
                 if (project.IsSubmission && project.SupportsCompilation)
                 {
                     cancellationToken.ThrowIfCancellationRequested();
 
                     // If we are referencing another project, store the link in the other direction
                     // so we walk across it later
-                    var compilation = await project.GetCompilationAsync(cancellationToken).ConfigureAwait(false);
-                    var previous = compilation.ScriptCompilationInfo.PreviousScriptCompilation;
+                    var compilation = await project.GetRequiredCompilationAsync(cancellationToken).ConfigureAwait(false);
+                    var previous = compilation.ScriptCompilationInfo?.PreviousScriptCompilation;
 
                     if (previous != null)
                     {
                         var referencedProject = solution.GetProject(previous.Assembly, cancellationToken);
```

Comment:
❔ Why not `GetRequiredProject` here?

Reply:
tbh, because i don't know if i expect the above to succeed.